### DISCUSSION TITLE
Connection points

### DISCUSSION THREAD

**Discussion by mjk-88** [jgraph/drawio#1976] [Ideas]
Hi All,

I use Diagrams a lot to make connection schematics but I struggle with the connection points on shapes.
I have thought about it and it would be great if there could be a connection point according to the lines in the grid.
So when I have a shape and make it bigger, there are automatically more waypoints.
I also snap all my lines to the grid so everything would look nice and clean.

Really hope this could be possible one day.

**Comment by davidjgraph** [comment]
Hold alt while dropping the connector to connect it anywhere on a shape.
